### DISCUSSION TITLE
color vs albedo_color RAW setting - bug?

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9967] [Editor]
Make a 3D mesh, add a material to it, click in inspector so you see albedo - color setting. Set inspector setting to property name styles - "RAW".

This still shows the albedo color as "color"

Actually, it should be "albedo_color" (as you can see from it's tooltip - you use albedo_color to access/change this in script).

Is this a bug or does RAW mean something else?

**Comment by Calinou** [comment]
The RAW option in the color picker affects the color scheme in the color picker only. It doesn't affect naming.

To change the property name style, use the icons in the top-right corner of thd inspector, or change it in the Editor Settings.

**Reply by shak2** [reply]
> The RAW option in the color picker affects the color scheme in the color picker only. It doesn't affect naming.
> 
> To change the property name style, use the icons in the top-right corner of thd inspector, or change it in the Editor Settings.

I mean the RAW in the property name style, not RAW in the color picker (I didn't even know color picker has a RAW!).

I still think this is a bug...
